src/main/java/org/jenkinsci/plugins/ewm/actions/ExwsAllocateActionImpl.java

```diff
@@ -23,12 +26,14 @@
     private final List<ExternalWorkspace> allocatedWorkspaces = new LinkedList<>();
```

Comment:
I'm fine with it. IIRC there was such proposal in one of previous PRs, but finally we decided it's not important due to the low number of workspaces.

But map really make sense since you're working on fingerprint integration